qemu/tests/sgx_multi_vms.py

```diff
@@ -0,0 +1,47 @@
+from virttest import error_context
+from virttest import env_process
+from virttest.utils_misc import verify_dmesg
+
+from provider.sgx import SGXHostCapability
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx multi vms boot test:
+    1. Check host sgx capability
+    2. Boot multi sgx VMs
+    3. Verify each VM sgx enabled in guest
+    4. Check each VM sgx qmp cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error_context.context("Boot four guests", test.log.info)
+    timeout = int(params.get_numeric("login_timeout", 240))
```

Comment:
we can remove int, i.e. timeout = params.get_numeric("login_timeout", 240), because int is the default numeric value type, see the function in utils_params: def get_numeric(self, key, default=0, target_type=int), fyi.